cpp/include/cudf/interop.hpp

```diff
@@ -126,23 +126,56 @@ struct column_metadata {
  *
  * @param input table_view that needs to be converted to arrow Table
  * @param metadata Contains hierarchy of names of columns and children
+ * @param stream CUDA stream used for device memory operations and kernel launches.
```

Comment:
```suggestion
 * @param stream CUDA stream used for device memory operations and kernel launches
```
Occurs in multiple places